src/arc.ts

```diff
@@ -165,6 +164,11 @@ export class Arc {
     return Observable.create(async (observer: Observer<ApolloQueryResult<any>>) => {
       Logger.debug(query.loc.source.body)
 
+      if (!apolloQueryOptions.fetchPolicy) {
+        // apolloQueryOptions.fetchPolicy = 'cache-and-network'
```

Comment:
can you remove this?

Reply:
i'm leaving it as a note, I need to refactor this part anyway #147